### DISCUSSION TITLE
How to get Legal notice

### DISCUSSION THREAD

**Discussion by defunty** [Shopify/storefront-api-feedback#128] [Help] [answered]
Hi.

How can I get shop's `Legal notice` with storefront API?

I can get another policy (Privacy policy, Terms of service, ...) from Shop Object.

This document don't refer to `Legal notice`.
https://shopify.dev/api/storefront/2022-07/objects/ShopPolicy

Can I get this or should I use Page as alternative?

**Comment by johncraigcole** [comment, marked as answer]
Hey @defunty legal information is typically captured with a custom page which can be found in our [docs](https://help.shopify.com/en/manual/checkout-settings/refund-privacy-tos#add-custom-pages-to-display-legal-information).

With that mind, assuming the custom or private app used for the Storefront API query has the `unauthenticated_read_content` scope you could query for a legal page that way.

```
query getLegalPage {
	page(handle: "legal-page") {
		title
		body
	}
}
```

**Reply by defunty** [reply]
@johncraigcole 
Thank you.

Is `Legal notice` is custom page?
I think it is store policy.

In [docs](https://help.shopify.com/en/manual/checkout-settings/refund-privacy-tos) 

> Adding store policies
> You can add or generate the following policies for your Shopify store:
> 
> Refund policy
> Privacy policy
> Terms of service
> Shipping policy
> Legal notice

Shop object have policies not only `Legal notice`.
Why is it?

**Reply by johncraigcole** [reply]
On the docs page if you scroll down you'll see that legal information is added via a page and not a policy on the `Settings->Policies` section of the Shopify admin. I'm not sure why the legal information is not available in the Policies settings in the admin, let me try and see if I can answer to that. Based on the help instructing you to create a custom page for legal information the page query I identified above would then be the only way to access that information via API.

**Reply by defunty** [reply]
OK, I will use custom page.
Thank you.
